source/deimos/openssl/opensslv.d

```diff
@@ -28,21 +120,26 @@ import deimos.openssl._d_util;
  */
 
 /* Version macros for compile-time API version detection */
-enum
-{
-    OPENSSL_VERSION_MAJOR   = 1,
-    OPENSSL_VERSION_MINOR   = 1,
-    OPENSSL_VERSION_PATCH   = 0,
-    OPENSSL_VERSION_BUILD   = 'h' - '`'
-}
+deprecated("Use `OpenSSLVersion.Major` instead, which has a correct value")
```

Comment:
Why not make these point to the new declarations?

Reply:
I wanted to avoid misleading users. The existing names match the C names. If anyone use them, they are probably doing it wrong. I also was not envisioning a major release, which is why I wanted to avoid a change in behavior. But regardless, I think having something that matches the C macro is a bit misleading.

Reply:
Could you please elaborate on why you think making these point to the values in the struct would be misleading? I would think that leaving them at their old values would be misleading, as they no longer represent the version of the OpenSSL that the bindings are targeting.

Reply:
If we're doing a major release, it's fine. If we were going for a minor release, it would change a constant that the user might (unwillingly / mistakenly) rely on.